element/index.js

```diff
@@ -82,11 +82,11 @@ export function renderToString( element ) {
 /**
  * Concatenate two or more React children objects
  *
- * @param  {...?Object} childrens Set of children to concatenate
- * @return {Array}                The concatenated value
+ * @param  {...?Object} childrenArguments Array of children arguments (array of arrays/strings/objects) to concatenate
+ * @return {Array}                				The concatenated value
```

Comment:
I diff the diff here should be spaces and not tabs, that way the text would be aligned?